tests/unit/channel_test.py

```diff
@@ -237,7 +237,7 @@ def test_buffer_frame(self):
     def test_process_frames_when_no_frames(self):
         # Not that this should ever happen, but to be sure
         c = Channel(None, None, {})
-        stub(c.dispatch)
+        expect(c.dispatch).times(0)
```

Comment:
You're welcome to whatever you'd like. I don't get much feedback on Chai usage, so it's good to know your perspective. I think I'll add a note to the documentation to call out how those are equivalent.